### ISSUE TITLE
Issue with doc build

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#347]
observed in 9d2586e76cc07e7435fe77b02313e241be73c192

<details><summary>Documentation build</summary>

~~~
/Users/julien/code/ivadomed/ivadomed/transforms.py:docstring of ivadomed.transforms.DilateGT:9: WARNING: Unexpected indentation.
/Users/julien/code/ivadomed/ivadomed/transforms.py:docstring of ivadomed.transforms.DilateGT:10: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/julien/code/ivadomed/ivadomed/transforms.py:docstring of ivadomed.transforms.Clahe:8: WARNING: Unexpected indentation.
/Users/julien/code/ivadomed/ivadomed/utils.py:docstring of ivadomed.utils.structurewise_uncertainty:5: WARNING: Unexpected indentation.
/Users/julien/code/ivadomed/ivadomed/utils.py:docstring of ivadomed.utils.segment_volume:9: WARNING: Unexpected indentation.
/Users/julien/code/ivadomed/docs/source/tutorials/one_class_segmentation_2d_unet.rst:154: WARNING: image file not readable: tutorials/../../images/sc_prediction.png
~~~

</details>

**Comment by lrouhier** [comment]
Will be fixed in #341

**Comment by jcohenadad** [comment]
now fixed (lost track where it was fixed)
